extensions/2.0/Vendor/GRIFFEL_bim_data/schema/node.properties.schema.json

```diff
@@ -0,0 +1,29 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
+    "title": "Node Properties and Type",
+    "type": "object",
+    "description": "Maps properties to nodes by node index. Used only when data is saved in a separate file.",
+    "allOf": [ { "$ref": "glTFProperty.schema.json" } ],
+    "properties": {
+        "node": {
```

Comment:
This bidirectional link looks very fragile. How are potential inconsistencies resolved?

Reply:
They are not. Node references a buffer where data should be found by the index of the node. If the buffer doesn't contain corresponding node data - than the node will have no data. If there are many data in the buffer with the same node ID - all should be read.